app/javascript/dashboard/helper/snoozeHelpers.js

```diff
@@ -66,3 +66,23 @@ export const snoozedReopenTime = snoozedUntil => {
   }
   return snoozedUntil ? format(date, 'd MMM, h.mmaaa') : null;
 };
+export const snoozedReopenTimeToTimestamp = snoozedUntil => {
+  return snoozedUntil ? getUnixTime(new Date(snoozedUntil)) : null;
+};
+export const shortenSnoozeTime = snoozedUntil => {
+  if (!snoozedUntil) {
+    return null;
+  }
+  const shortenTime = snoozedUntil
```

Comment:
Instead of chaining multiple replace calls, you can use regular expressions replace all occurrences in one go.

```
 const shortenTime = snoozedUntil.replace(
    /( minutes?| hours?| days?| months?| years?)/g,
    (match, unitMatch) => {
      if (unitMatch === ' minutes' || unitMatch === ' minute') return 'm';
      if (unitMatch === ' hours' || unitMatch === ' hour') return 'h';
      if (unitMatch === ' days' || unitMatch === ' day') return 'd';
      if (unitMatch === ' months' || unitMatch === ' month') return 'mo';
      if (unitMatch === ' years' || unitMatch === ' year') return 'y';
      return match;
    }
  );
  ```